src/debugger.fs

```diff
@@ -63,8 +63,12 @@ module Program =
     open FSharp.Reflection
 
     let inline private duName (x:'a) =
-        match FSharpValue.GetUnionFields(x, typeof<'a>) with
-        | case, _ -> case.Name
+        let t = typeof<'a>
+        if (isNull t?cases)
```

Comment:
Instead of relying in the internal JS representation of F# unions by Fable (which may change in the future), you can just use `FSharpType.IsUnion`: https://github.com/fable-compiler/Fable/blob/e2ce9884390d95d223044e666c7c251c7392b428/tests/Main/ReflectionTests.fs#L385

Latest Fable.Core also includes some reflection helpers: https://github.com/fable-compiler/Fable/blob/e2ce9884390d95d223044e666c7c251c7392b428/src/Fable.Core/Fable.Core.Util.fs#L26-L32